scripts/cli.py

```diff
@@ -152,6 +152,12 @@ class Constants:
         BOARD_ALIASES = {'dk':     BOARD_DK,
                          'mesh':   BOARD_MESH,
                          'dongle': BOARD_DONGLE}
+        USER_DTS_OVERLAY_FILE = {BOARD_DK:     'nrf52840_pca10056-overlay.dts',
```

Comment:
You can do a patch improving this. Make something like that:
```
BOARDS = [BOARD_DK, BOARD_MESH, BOARD_DONGLE]
USER_DTS_OVERLAY_FILE =  {board: board+'-overlay.dts' for board in BOARDS }
USER_DTS_OVERLAY_FILE_ALIASES = {board: board+'-overlay.dts' for board in BOARD_ALIASES }
```
This will help to change just the array with the supported boards :)

Reply:
Done.